python/lsst/ts/m2gui/controltab/tab_config_view.py

```diff
@@ -33,7 +37,233 @@ class TabConfigView(TabDefault):
+        layout.addWidget(self._label_file_configuration)
+        layout.addWidget(self._label_file_version)
+        layout.addWidget(self._label_file_control_parameters)
+        layout.addWidget(self._label_file_lut_parameters)
+
+        layout.addSpacing(space_between_sections)
+
+        label_power_supply = create_label(
+            name="Power Supply Monitoring", point_size=point_size_section, is_bold=True
+        )
+        layout.addWidget(label_power_supply)
+
+        label_power_motor = create_label(
+            name="Motor", point_size=point_size_subsection, is_bold=True
+        )
+        layout.addWidget(label_power_motor)
+        layout.addWidget(self._label_power_warning_motor)
+        layout.addWidget(self._label_power_fault_motor)
+        layout.addWidget(self._label_power_threshold_motor)
+
+        label_power_communication = create_label(
+            name="Communication", point_size=point_size_subsection, is_bold=True
+        )
+        layout.addWidget(label_power_communication)
+        layout.addWidget(self._label_power_warning_communication)
+        layout.addWidget(self._label_power_fault_communication)
+        layout.addWidget(self._label_power_threshold_communication)
+
+        layout.addSpacing(space_between_sections)
+
+        label_in_position_flag = create_label(
+            name="In-Position Flag", point_size=point_size_section, is_bold=True
+        )
+        layout.addWidget(label_in_position_flag)
+        layout.addWidget(self._label_in_position_axial)
+        layout.addWidget(self._label_in_position_tangent)
+        layout.addWidget(self._label_in_position_sample)
+
+        layout.addSpacing(space_between_sections)
+
+        label_communication_timeout = create_label(
+            name="Communication Timeout", point_size=point_size_section, is_bold=True
+        )
+        layout.addWidget(label_communication_timeout)
+        layout.addWidget(self._label_timeout_sal)
+        layout.addWidget(self._label_timeout_crio)
+        layout.addWidget(self._label_timeout_ilc)
+
+        layout.addSpacing(space_between_sections)
+
+        label_miscellaneous = create_label(
+            name="Miscellaneous", point_size=point_size_section, is_bold=True
+        )
+        layout.addWidget(label_miscellaneous)
+
+        label_inclinometer = create_label(
+            name="Inclinometer", point_size=point_size_subsection, is_bold=True
+        )
+        layout.addWidget(label_inclinometer)
+        layout.addWidget(self._label_misc_range_angle)
+        layout.addWidget(self._label_misc_diff_enabled)
+
+        label_temperature = create_label(
+            name="Temperature", point_size=point_size_subsection, is_bold=True
+        )
+        layout.addWidget(label_temperature)
+        layout.addWidget(self._label_misc_range_temperature)
+
+        return layout
+
+    def _set_signal_config(self, signal_config):
+        """Set the config signal with callback function.
```

Comment:
Does that really need to be a method? This just obfuscates the code. You don't gain any benefit from separating that to a function, you are just adding lines. Having single line right in __init__ is much better.

Reply:
I prefer to have an abstract function to indicate this `signal_config` can come from any object. This might not gain any benefit but it does not hurt anything as well. Thanks!